packages/kit/src/exports/public.d.ts

```diff
@@ -642,6 +643,21 @@ export interface KitConfig {
 		 * @since 1.16.0
 		 */
 		handleEntryGeneratorMismatch?: PrerenderEntryGeneratorMismatchHandlerValue;
+		/**
+		 * How to respond when a route is marked as prerenderable but has not been prerendered.
+		 *
+		 * - `'fail'` — fail the build
+		 * - `'ignore'` - silently ignore the failure and continue
+		 * - `'warn'` — continue, but print a warning
+		 * - `(details) => void` — a custom error handler that takes a `details` object with a `routes` property which contains all routes that haven't been prerendered. If you `throw` from this function, the build will fail
+		 *
+		 * The default behavior is to fail the build. This may be undesireable when you know that some of your routes may never be reached under certain
```

Comment:
```suggestion
		 * The default behavior is to fail the build. This may be undesirable when you know that some of your routes may never be reached under certain
```